scripts/create_deploy_key.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Colors
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+mkdir -p ./tmp
+
+if [[ -f "./tmp/github_deploy_key" ]]; then
+  echo -e "${RED}Deploy key already exists${NC}"
+  rm ./tmp/github_deploy_key
+fi
+
+if [[ -f "./tmp/github_deploy_key.pub" ]]; then
+  echo -e "${RED}Public key already exists${NC}"
+  rm ./tmp/github_deploy_key.pub
+fi
+
+ssh-keygen \
+  -t rsa \
+  -b 4096 \
+  -C "melanie@deerhide.run" \
```

Comment:
Shouldn't this use the hostname (fact or env var)?

Reply:
it's just for generation manually and setup it on github